util/exercise/src/lib.rs

```diff
@@ -235,7 +235,7 @@ pub fn generate_test_function(case: &Value, use_maplit: bool) -> Result<String>
         use Value::*;
         match comments {
             Array(cs) => cs
-                .iter()
+                .par_iter()
```

Comment:
We're iterating over the number of comments for a single test, here. This will not be a helpful speedup.